specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2425,6 +2494,18 @@
       "description": "AAD (Azure Active Directory) data connector properties.",
       "type": "object"
     },
+    "AADCheckRequirements": {
```

Comment:
>AADCheckRequirements [](start = 5, length = 20)

you may want to use discriminator pattern. i suppose we want the clients to use any of these objects (that inherit from DataConnectorsCheckRequirements) with the newly added POST.
This object has additional property of DataConnectorTenantId

Reply:
It is not the only thing that inherits from DataConnectorsCheckRequirements that has DataConnectorTenantId.
I added discriminators to anything that inherits from DataConnectorsCheckRequirements. Thanks